File: PYTHON/barista.py
from pegawai import Pegawai

class Barista(Pegawai):
    def __init__(self, nik, nama, jenis_kelamin, id_pegawai, departemen, tahun_masuk, shift_kerja, menu_andalan, foto_produk):
        super().__init__(nik, nama, jenis_kelamin, id_pegawai, departemen, tahun_masuk)
        self.shift_kerja = shift_kerja
        self.menu_andalan = menu_andalan
        self.foto_produk = foto_produk

    def get_shift_kerja(self): return self.shift_kerja
    def get_menu_andalan(self): return self.menu_andalan
    def get_foto_produk(self): return self.foto_produk

File: JAVA/Barista.java
public class Barista extends Pegawai {
    private String shiftKerja;
    private String menuAndalan;
    private String fotoProduk;

    public Barista() {}

    public Barista(String nik, String nama, String jenisKelamin, String idPegawai, String departemen, int tahunMasuk, String shiftKerja, String menuAndalan, String fotoProduk) {
        super(nik, nama, jenisKelamin, idPegawai, departemen, tahunMasuk); // Memanggil constructor parent (Pegawai)
        this.shiftKerja = shiftKerja;
        this.menuAndalan = menuAndalan;
        this.fotoProduk = fotoProduk;
    }

    public String getShiftKerja() { return shiftKerja; }
    public String getMenuAndalan() { return menuAndalan; }
    public String getFotoProduk() { return fotoProduk; }
}